client/components/app.jsx

```diff
@@ -9,24 +9,34 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      restaurantId: 1001,
+      restaurantId: 1001, 
       timeSlots: [],
       partySize: 2,
+      partySizeMax: 15,
       date: "2018-05-31",
-      time: "5:00 am",
+      time: "5:00 PM",
       pickTime: "",
       clicked: false,
```

Comment:
Super small detail which is definitely not necessary, most variables which represent boolean state are written as "isClicked".